### PR TITLE
bpo-40670: Improve documentation for timeit function

### PR DESCRIPTION
**Pull request by edison12a**
https://bugs.python.org/issue40670

### CODE REVIEW

File: Lib/timeit.py [hunk 1]
```diff
@@ -229,7 +229,27 @@ def autorange(self, callback=None):
 
 def timeit(stmt="pass", setup="pass", timer=default_timer,
            number=default_number, globals=None):
-    """Convenience function to create Timer object and call timeit method."""
+    """Convenience function to create Timer object and call timeit method.
+
+    Args:
+        stmt (str): The code to be excuted. This can have multiple statements
+            separated by ; and \n.
```
Comment (sethtroisi) [inline Lib/timeit.py:236]: should this be "seperated by ; **or** \n."?

File: Lib/timeit.py [hunk 2]
```diff
@@ -229,7 +229,27 @@ def autorange(self, callback=None):
...
+            executed within that namespace (as opposed to inside timeit's
+            namespace).
+
+    Returns:
+        float: The time it takes to execute the statement under given
+            conditions.
+
+    Raises:
+        IndentationError: If 'stmt' is an empty string.
+    """
```
Comment (jacobtylerwalls) [inline Lib/timeit.py:252]: No longer true after 557b9a5

File: Lib/timeit.py [hunk 3]
```diff
@@ -229,7 +229,27 @@ def autorange(self, callback=None):
 
 def timeit(stmt="pass", setup="pass", timer=default_timer,
            number=default_number, globals=None):
-    """Convenience function to create Timer object and call timeit method."""
+    """Convenience function to create Timer object and call timeit method.
+
+    Args:
+        stmt (str): The code to be excuted. This can have multiple statements
```
Comment (jacobtylerwalls) [inline Lib/timeit.py:235]: "executed"

File: Doc/library/timeit.rst [hunk 4]
```diff
@@ -98,12 +98,10 @@ The module defines three convenience functions and a public class:
 
    Class for timing execution speed of small code snippets.
 
-   The constructor takes a statement to be timed, an additional statement used
-   for setup, and a timer function.  Both statements default to ``'pass'``;
-   the timer function is platform-dependent (see the module doc string).
-   *stmt* and *setup* may also contain multiple statements separated by ``;``
-   or newlines, as long as they don't contain multi-line string literals.  The
-   statement will by default be executed within timeit's namespace; this behavior
+   The constructor takes suite of statement to be timed, an additional suite
```
Comment (jacobtylerwalls) [inline Doc/library/timeit.rst:101]: "a suite"

**Comment by iritkatriel** [comment]
https://bugs.python.org/issue40670 is closed. What is the status of this PR?

**Comment by rhettinger** [comment]
Personally, I don't find this to be a helpful improvement.  If someone else thinks otherwise; feel free to reopen.

**Pull request closed without merging**